### PR TITLE
Recyclebinfix

### PR DESCRIPTION
**Pull request by emilwestergren**
* Fixed bug when clearing bin
* Added small test

@marcusnaslund peer review

### CODE REVIEW

File: source/collections/RecycleBin.ooc [hunk 1]
```diff
@@ -22,6 +22,9 @@ RecycleBin: class <T> {
 	//TODO: Replace with SynchronizedVectorList
 	_list: VectorList<T>
 	_size: Int
+	count ::= this _list count
+	full ::= this count == this _size
+	empty ::= this _list empty
```
Comment (marcusnaslund) [inline source/collections/RecycleBin.ooc]: Should these be `isFull` and `isEmpty` from now on?

Comment (emilwestergren) [inline source/collections/RecycleBin.ooc]: I guess, unless we want to change everything at once.

Comment (marcusnaslund) [inline source/collections/RecycleBin.ooc]: Preferably not.

**Comment by emilwestergren** [comment]
Updated.

**Comment by marcusnaslund** [comment]
Don't forget magic.

**Comment by emilwestergren** [comment]
Updated.

**Pull request merged**